doc/nrf/releases/release-notes-latest.rst

```diff
@@ -91,6 +91,36 @@ The following list summarizes the most important changes inherited from upstream
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
 
+* Storage and filesystems:
+
+  * A possible NULL pointer dereference when using any of the ``fs_`` functions has been fixed and will not return an error code instead.
```

Comment:
Should this be "will now" instead of "will not"?

Reply:
Yes, thanks.